matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -1108,30 +1271,16 @@ var PhenoTips = (function (PhenoTips) {
         return buttonHTML;
     },
 
-    _getNotified : function(match)
-    {
-        var td = '<td style="text-align: center" name="notified">';
-        td += this._getMarkNotifiedButtonHTML(match);
-        td += '</td>';
-        return td;
-    },
-
-    _getMarkNotifiedButtonHTML : function(match)
+    _getMarkUserContactedHTML : function(match)
```

Comment:
"getMark____" is a very confusing word combination, something like `getManualContactButtonHTML()` would be better

Reply:
I'd leave it like that